packages/jellyfish-api-core/__tests__/category/governance/createCfp.test.ts

```diff
@@ -0,0 +1,170 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+import { ProposalStatus, ProposalType } from '../../../src/category/governance'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+describe('Governance', () => {
+  const container = new MasterNodeRegTestContainer(undefined, 'defi/defichain:HEAD-a61f615', undefined, ['-fortcanningheight=7'])
```

Comment:
The existing code definitely allow us to pass in param. This is one of the way

```
 class MyContainer extends MasterNodeRegTestContainer {
    public getCmd (opts: StartOptions): string[] {
      return [
        ...super.getCmd(opts),
        '-fortcanningheight=7'
      ]
    }
  }
  const container = new MyContainer(undefined, 'defi/defichain:HEAD-a61f615', undefined, )

```

Reply:
Indeed, I did a Governance container as it will be needed for following PRs